its/polyglot-scala/pom.xml

```diff
@@ -15,7 +15,9 @@
     <artifactId>its</artifactId>
     <version>0.7.3-SNAPSHOT</version>
   </parent>
+
   <artifactId>polyglot-scala</artifactId>
+  <name>Polyglot :: Scala ITs</name>
```

Comment:
What does the "ITs" stand for?

I read it as "Integration Tests" but the module clearly contains more than just tests, it also contains the integration itself.